btcnode.adoc

```diff
@@ -0,0 +1,95 @@
+= Bitcoin Core Nodes
+
+Bisq contributors run a federation of Bitcoin Core nodes to ensure Bisq connects to nodes with consistent Bitcoin Core implementations.
+
+
+== Introduction
+
+In the past, Bisq connected to arbitrary Bitcoin Core nodes, but the prospect of the SegWit2x hard fork in Fall 2017 made it clear this was not ideal—in the case of a hard fork, it would be impossible for Bisq to tell which fork a connected node was running. The SegWit2x hard fork never happened, but if it did, there was a chance transactions would execute on an undesirable fork, and even worse, a chance that transactions would execute on different forks.
+
+Hence the decision to run a federation of Bitcoin nodes known to run the same Bitcoin Core implementation and configuration—Bisq can expect all bitcoin nodes it connects to enforce consistent block validation rules.
+
+An important additional benefit of this setup is added user privacy: bloom filters in bitcoinj are not implemented very well, so connecting only to nodes run by trusted Bisq contributors avoids monitoring from arbitrary public nodes.
```

Comment:
I would rephrase it so something like that:
bloom filters carry serious privacy flaws when connecting to arbitrary bitcoin nodes as chain analysis companies try to spy on SPV wallets. Connecting only to nodes run by trusted Bisq contributors avoids such monitoring attempts from arbitrary public nodes.